packages/eslint-plugin-jest/src/index.js

```diff
@@ -20,6 +21,7 @@ module.exports = {
         'jest/no-disabled-tests': 'warn',
         'jest/no-focused-tests': 'error',
         'jest/no-identical-title': 'error',
+        'jest/no-to-be-float': 'error',
```

Comment:
I don't think it should be `error` by default. `0.1 + 0.2` is a rare rounding error. I want to assert that `3 / 2 === 1.5` sometimes.